packages/desktopjs/src/registry.ts

```diff
@@ -18,11 +18,11 @@ export class ContainerRegistration {
     create: (options?: any) => Container;
 }
 
-const registeredContainers: Map<string, ContainerRegistration> = new Map();
+const registeredContainers: {id: string, registration: ContainerRegistration}[] = [];
 
 /** Clears all container registrations. */
 export function clearRegistry() {
-    registeredContainers.clear();
+    registeredContainers.splice(0, registeredContainers.length - 1);
```

Comment:
Should we not keep the semantic of clear?

Reply:
I am not sure what you mean. registeredContainers was a Map, so clear just emptied the map, same as splice for Array.

Reply:
```
var months = ['Jan', 'March', 'April', 'June'];
months.splice(0, months.length - 1);
console.log(months);
```

This returns ["June"] instead of [].

Reply:
oh yeah, this was fixed in subsequent commit.